anda/misc/uxn/stable/uxn.spec

```
@@ -0,0 +1,42 @@
+# uxn stable has no update script because this version
+# is permanently frozen. A nightly will be added once the
+# sourcehut versioning script is done.
+
+%define debug_package %nil
+
+Name:           uxn
+Version:        1.0
+Release:        1%?dist
+Summary:        An emulator for the Varvara virtual machine
+URL:            https://100r.ca/site/%{name}.html
+Source0:        https://git.sr.ht/~rabbits/%{name}/archive/%{version}.tar.gz
+License:        MIT
+BuildRequires:  anda-srpm-macros SDL2-devel gcc
```

```suggestion
BuildRequires:  SDL2-devel gcc
```
No anda specific macros are being used here, so `anda-srpm-macros` isn't needed.